src/main/features/ccj/views/paid-amount.njk

```diff
@@ -0,0 +1,67 @@
+{% extends "layout.njk" %}
+
+{% set title = 'Money claim' %}
+
+{% from "includes/macros/form.njk" import saveAndContinueButton, csrfProtection, textInput, radioOption, errorSummary %}
+
+{% block content %}
+<div class="grid-row">
+  <div class="column-two-thirds">
+
+    <h1 class="heading-large">
+      <span class="heading-secondary">{{ t("County Court Judgment") }} </span>
+
+      {{ t("Has the defendant paid some of the amount owed?") }}
+    </h1>
+
+    <form method="post" novalidate>
+
+      {{ csrfProtection(csrf) }}
+
+      {% set error = form.errorFor('option') %}
+      {% set selection = form.valueFor('option') %}
+      <div class="form-group {% if error %} form-group-error {% endif %}">
+        <fieldset>
+          <legend class="visually-hidden">Has the defendant paid some of the amount owed?</legend>
+
+          {{
```

Comment:
radioOption should never be used directly on a page, use radioGroup.
See https://github.com/hmcts/cmc-citizen-frontend/pull/16/files#diff-515e6f19e3ea26bc5e9ed26ab69f5eefR22